netbox/circuits/models.py

```diff
@@ -265,7 +269,12 @@ class CircuitTermination(CableTermination):
 
     class Meta:
         ordering = ['circuit', 'term_side']
-        unique_together = ['circuit', 'term_side']
+        constraints = [
+            models.UniqueConstraint(
+                fields=['circuit', 'term_side'],
+                name='circuittermination_circuit_termside_key'
```

Comment:
Keep an eye on the field names (`term_side`, not `termside`). I realize this seems pedantic but it'll be important if we do ever start automatically generating constraint names.

Reply:
The problem with this approach is that it is ambiguous (because `_` is the separator between fields and this could be read as `term` and `side` as separate fields); I think either way has its pros and cons—as this one clearly also shuffles things around—, but ultimately I don’t care either way.

Reply:
The ambiguity is acceptable in this case; Django's stock naming has the same issue. It's more important that we preserve the ability to programmatically reproduce the constraint names.

Reply:
I renamed things. I think I caught everything, but of course I’m not sure!